docs/mkdocs/developer/index.md

```diff
@@ -70,29 +73,31 @@ status:
 ```
 
 ## In-cluster builds
+
 Virtually any Kubernetes distribution ships with its own kernel.
 It is thus challenging for a kernel module vendor to make DriverContainer images available for all kernels available
 in all Kubernetes distributions out there (or even for a subset of them).
 
-KMM supports in-cluster builds of DriverContainer images when those are not made available by the vendor.
-The in-cluster build system is able to build an image from any Git repository that contains a `Dockerfile`.
-It can then push said image to a user-provided repository, optionally using authentication provided in a Kubernetes secret.
+KMM supports in-cluster builds of DriverContainer images when those are not made available by the vendor. The in-cluster build system is able to build an image from any Git repository that contains a `Dockerfile`. It can then push said image to a user-provided repository, optionally using authentication provided in a Kubernetes secret.
+
+**Optional:** on upstream Kubernetes, we may want to deploy an in-cluster registry to host in-cluster built images.
```

Comment:
How about adding a an example using minikube to the doc? Here is a reference on how to do it:
* https://github.com/kubernetes-sigs/kernel-module-management/blob/ee344d6183cb16d649d7df592242cfff01c52610/.github/actions/create-minikube-cluster/action.yml#L30
* https://github.com/kubernetes-sigs/kernel-module-management/blob/ee344d6183cb16d649d7df592242cfff01c52610/.github/workflows/e2e.yaml#L43-L46

Reply:
Raised https://github.com/kubernetes-sigs/kernel-module-management/issues/243 for this